src/schema/schema_create.c

```diff
@@ -90,8 +97,51 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
     /* Sanity check the allocation size. */
     WT_ERR(__wt_direct_io_size_check(session, filecfg, "allocation_size", &allocsize));
 
-    /* Create the file. */
-    WT_ERR(__wt_block_manager_create(session, filename, allocsize));
+    /*
+     * If we are importing an existing object rather than creating a new one, there are two possible
+     * scenarios. Either (1) the file configuration string from the source database metadata is
+     * specified in the input config string, or (2) the import.repair option is set and we need to
+     * reconstruct the configuration metadata from the file.
+     */
+    if (import) {
+        import_repair =
+          __wt_config_getones(session, config, "import.repair", &cval) == 0 && cval.val != 0;
+        if (!import_repair) {
+            if (__wt_config_getones(session, config, "import.file_metadata", &cval) == 0 &&
+              cval.len != 0) {
+                /*
+                 * The string may be enclosed by delimiters (e.g. braces, quotes, parentheses) to
```

Comment:
is it a "may be"? or we are certain that there will be delimiters. Given that we discard first and last unconditionally i am assuming it is certain.

Reply:
I'm assuming that the "may be" is referring to the fact that this only applies nested structures (`WT_CONFIG_ITEM_STRUCT`).

@tammybailey is that right?

Reply:
Yes. Does it add clarity to say "Discard the first and last characters in this case"?

Reply:
3666e15